internal/server/http/user/handlers.go

```diff
@@ -26,18 +26,25 @@ type UsersUsecases interface {
 	Authenticate(ctx context.Context, email, password string) (id string, err error)
 }
 
+type PasswordUsecases interface {
+	SendRestorePasswordCode(ctx context.Context, email string) (string, error)
+	DeleteCode(ctx context.Context, email string) error
+	Authenticate(ctx context.Context, pc entity.PasswordCode) (string, error)
+}
+
 const idKey = "id"
 const tokenTime = time.Minute * 15
 
 // Handlers represent a user handlers.
 type Handlers struct {
-	usecases UsersUsecases
-	logger   *zap.Logger
+	userCases UsersUsecases
+	paswCases PasswordUsecases
```

Comment:
why do we need  it here ? can we have separate handlers ?

Reply:
@igkostyuk I will think about it. If we are going to use it as other service, I will move it out soon.

Reply:
for this change user handlers can't exist without paswCases